ServerMonitoringSystem.SignalRHub/Program.cs

```diff
@@ -0,0 +1,27 @@
+﻿using ServerMonitoring.SignalRHub.Hubs;
+
+var builder = WebApplication.CreateBuilder(args);
+
+builder.Services.AddSignalR();
+builder.Services.AddControllers();
+builder.Services.AddEndpointsApiExplorer();
+builder.Services.AddSwaggerGen();
+
+var app = builder.Build();
+
+if (app.Environment.IsDevelopment())
+{
+    app.UseSwagger();
+    app.UseSwaggerUI();
+}
+
+app.UseRouting();
+app.UseAuthorization();
+
+app.UseEndpoints(endpoints =>
+{
+    endpoints.MapControllers();
+    endpoints.MapHub<AlertHub>("/hub/alerts");
```

Comment:
i think this should be moved out and mapped like this:

`app.MapHub<AlertHub>("/hub/alerts");`

if you are not using mvc.